tests/Doctrine/Tests/ORM/Tools/EntityGeneratorTest.php

```diff
@@ -224,6 +224,26 @@ public function newInstance($metadata)
         return new $metadata->name;
     }
 
+    /**
+     * @group GH-6314
+     */
+    public function testEmbeddedEntityWithColumnPrefix()
+    {
+        $testMetadata = $this->generateTestEmbeddableFixture();
+        $isbnMetadata = $this->generateIsbnEmbeddableFixture(['testEmbedded' => $testMetadata], 'prefix');
```

Comment:
Please use a random value for the prefix passed in here - makes it harder to escape checks